### DISCUSSION TITLE
Fork doesn't work!

### DISCUSSION THREAD

**Discussion by oliveroing** [mazid1/ngx-mzd-timeline#9] [Ideas]
Hi, me again! I'm trying to fork the project. I added a new functionality that allows you to decide what side has to take each content. I pushed my fork branch and when I try to change the dependences in my project, it doesn't work. I get an ERROR. Do you know what can I do to get mi library-form working ? Thank you!

**Comment by oliveroing** [comment]
![Captura de Pantalla 2021-05-31 a la(s) 16 59 34](https://user-images.githubusercontent.com/17765949/120235978-98f04000-c231-11eb-85b8-ef8b975380bf.png)

**Reply by mazeduldev** [reply]
If it is your separate project where you are using `ngx-mzd-timeline` by installing through `npm` command then you should import it this way:
```
import { MzdTimelineModule } from 'ngx-mzd-timeline';
```
  
If it is a fork of the original repository and you are importing it into the demo app then you should import it this way:
```
import { MzdTimelineModule } from 'projects/ngx-mzd-timeline/src/public-api';
```
Your import statement in line number 19 is not correct.

**Reply by oliveroing** [reply]
OK, thank you! I will try that. Do I need to do something extra? Like build the project into dist or config a prepare/post-install script for example ?
